File: client/src/main/java/material/com/top/PageConfig.java
package material.com.top;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by zjl on 16/7/6.
 */
public class PageConfig {
    private static List<String> pageTitles = new ArrayList<String>();

    public static List<String> getPageTitles() {
        pageTitles.clear();
        pageTitles.add("首页");
        pageTitles.add("提交干货");
        return pageTitles;
    }

    private static final String NewFragment = "material.com.news.ui.NewFragment";
    public static final String AllNewsFragment = "material.com.news.ui.AllNewsFragment";
    public static final String SubmitFragment = "com.cangwang.submit.ui.SubmitFragment";

    public static String[] fragmentNames = {
            AllNewsFragment,
            SubmitFragment,
            NewFragment,
            NewFragment
    };
}

File: client/src/main/java/material/com/top/PageConfig.kt
package material.com.top

import java.util.ArrayList

/**
 * Created by cangwang on 16/7/6.
 */
object PageConfig {
    private val pageTitles = ArrayList<String>()

    fun getPageTitles(): List<String> {
        pageTitles.clear()
        pageTitles.add("首页")
        pageTitles.add("提交干货")
        return pageTitles
    }

    private val NewFragment = "material.com.news.ui.NewFragment"
    val AllNewsFragment = "material.com.news.ui.AllNewsFragment"
    val SubmitFragment = "com.cangwang.submit.ui.SubmitFragment"

    var fragmentNames = arrayOf(AllNewsFragment, SubmitFragment, NewFragment, NewFragment)
}
